src/auth/oauth2client.ts

```diff
@@ -20,6 +20,7 @@ import {
 } from 'gaxios';
 import * as querystring from 'querystring';
 import * as stream from 'stream';
+import * as formatEcdsa from 'ecdsa-sig-formatter';
```

Comment:
we would need to add `ecdsa-sig-formatter` as a new dependency in `package.json`:

```
npm i ecdsa-sig-formatter --save
```

Reply:
I thought this was a built-in library, similar to `crypto`. I guess i was wrong :P

Reply:
done!